src/sw.prod.js

```diff
@@ -16,28 +16,51 @@
  *  limitations under the License
  *
  */
+/* global workbox */
+
 workbox.skipWaiting();
 workbox.clientsClaim();
 
-/* global workbox */
+/**
+ * Precache (cache ahead of time)
+ */
 
-// Precache static assets.
+// Precache assets generated by workbox-webpack-plugin
 workbox.precaching.precacheAndRoute(self.__precacheManifest || []);
 
-// Cache at runtime: partial navigation routes (fragments).
-const offlineNavigationResponse = `It seems you don't have
-  internet access at the moment.
-  Please try again later.`;
+/**
+ * Handle navigation requests
+ */
+
+workbox.routing.registerNavigationRoute('/app-shell');
```

Comment:
Add a link to the Workbox documentation here as this is the most magical part in my opinion. https://developers.google.com/web/tools/workbox/modules/workbox-routing

Reply:
A better comment too that just summarises the behaviour that's defined as a result of this call.